cmd/kubeadm/app/cmd/token.go

```diff
@@ -37,7 +37,7 @@ import (
 	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
 	tokenutil "k8s.io/kubernetes/cmd/kubeadm/app/util/token"
 	bootstrapapi "k8s.io/kubernetes/pkg/bootstrap/api"
-	"k8s.io/kubernetes/pkg/kubectl"
+	"k8s.io/kubernetes/pkg/printers"
```

Comment:
Why top level and not under `kubectl`?  I'm assuming its because you anticipate them moving into the apiserver in a couple releases?